content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/2-postgresql-db.md

```diff
@@ -0,0 +1,147 @@
+
+
+* Clients must set `sslMode=require` or greater as the server instance requires encryption for all connections. Review the [Postgres SSL documentation](https://www.postgresql.org/docs/current/libpq-ssl.html) to learn more about the SSL modes.
+
+
+# Add Postgres to an App Profile
+
+
+## Prerequisite
+
+A Spectro Cloud [account](https://www.spectrocloud.com/get-started/)
+
+## Enablement
+
+You can use the following steps to learn how to add Postgres to your app profile.
+
+1. Log in to [Palette](console.spectrocloud.com).
+
+
+2. On the right side of the window, click the **User Menu** to expand it and select **Switch to App Mode**.
+
+
+3. Navigate to the left **Main Menu** and click on **App Profiles** to create a [new App Profile](/devx/app-profile/create-app-profile/). Provide the following basic information and click **Next**.
+
+|         **Parameter**   | **Description**  |
+|-------------------------|-----------------|
+|Application Profile Name | A custom name for the App Profile.|
+|Version (optional)       | The default value is 1.0.0. You can create multiple versions of an App Profile using the format **`major.minor.patch`**.
+|Description (optional)   | Description of the App Profile. | 
+|Tag (optional)           | Assign tags to the app profile.|
+
+
+4. Select **Postgres** from the database services and start the configuration.
+  
+
+5. Provide the following information to the wizard:
+
+  * **Name:** The database service name. You can use the auto-generated name or create a custom name.
+
+
+  * **Username:** The user name for database access control. 
+
+
+  * **Password:** Security password for the DB service.
+
+<InfoBox>
+
+
+You can use the default system-generated password. If the default password is used, it can be retrieved from the Postgres secret that is created for the password. Review the [Database Password](/devx/app-profile/services/service-listings/postgresql-db/#database-password) section for guidance.
+
+</InfoBox>
+
+
+  * **Database Volume Size (GiB):** Select the volume size for the database. Ensure you stay within the storage amount available in the cluster group and virtual clusters. 
+
+  * **Version: **Select the version from the **Version** **drop-down Menu**. The following are the Palette supported PostgreSQL versions:
+
+    * 14
+   
+6. Click on **Save Changes**.
+
+## Validation
+
+* To validate that your database service is in the app profile, navigate to the **App Profiles** page, where all your app profiles are listed. Click on the app profile you wish to review the service layers. The following screen displays the different service layers that make up the app profile. Ensure MySQL is an available service layer.
+
+
+* Validate the services from the App page after app deployment. First, navigate to the **App** page, where all your apps are listed. Then, click the **App Name** to see the service layers. The color code in the app profile box shows the status of the service deployment.
```

Comment:
```suggestion
* Validate the services from the App page after app deployment. First, navigate to the **Apps** page, where all your apps are listed. Then, select the app to display the service layers. The color code in the app profile box shows the status of the service deployment.
```

Reply:
No boldface for **App Name** because the term **App Name** isn't a UI element. 

I might've missed this in one of the other files.